src/WebJobs.Extensions.CosmosDB/Bindings/CosmosDBAsyncCollector.cs

```diff
@@ -10,13 +10,13 @@
 using Microsoft.Azure.Documents.Client;
 using Newtonsoft.Json.Linq;
 
-namespace Microsoft.Azure.WebJobs.Extensions.DocumentDB
+namespace Microsoft.Azure.WebJobs.Extensions.CosmosDB
```

Comment:
This is one that I think we should avoid - I know the name has changed, but unfortunately this will break customers. It would be nice if we could have the old namespace as well as the new cosmosdb namespace & classes